src/main/java/com/zufar/onlinestore/cart/repository/ShoppingSessionItemRepository.java

```diff
@@ -15,4 +15,6 @@ public interface ShoppingSessionItemRepository extends JpaRepository<ShoppingSes
             nativeQuery = true)
     ShoppingSessionItem updateProductsQuantityInShoppingSessionItem(@Param("shopping_session_item_id") UUID shoppingSessionItemId,
                                                                     @Param("products_quantity_change") Integer productsQuantityChange);
+    @Query(value = "DELETE FROM shopping_session_item WHERE id = :shopping_session_item_id RETURNING *", nativeQuery = true)
```

Comment:
sql query `RETURNING *` returns the rowQuantity which were updated with this sql query. Please change the return type of this method to Integer not ShoppingSessionItem

Reply:
fixed